applications/java/deployment/eks_hpa.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: autoscaling/v2
```

Comment:
@omar-faruk01 @YogeshBhatiaAWS 
i see no reason to specify `eks_` in front of these kubernetes manifest files, considering that these are only used for the hybrid cluster which is on-prem.
I will merge for now, then we should rename here and in the workshop content.